claat/newparser/md/md_test.go

```diff
@@ -0,0 +1,169 @@
+package md
+
+import (
+	"bytes"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/googlecodelabs/tools/claat/types"
+	"golang.org/x/net/html"
+	"golang.org/x/net/html/atom"
+)
+
+func TestParse(t *testing.T) {
```

Comment:
It would be nice to have a case or a separate test that exercises the invalid metadata logic.